src/SLIP10Node.ts

```diff
@@ -299,11 +320,17 @@ export class SLIP10Node implements SLIP10NodeInterface {
         privateKey,
         curveObject.privateKeyLength,
       );
+
       assert(
         curveObject.isValidPrivateKey(privateKeyBytes),
         `Invalid private key: Value is not a valid ${curve} private key.`,
       );
 
+      const trustedPublicKey =
+        guard === PUBLIC_KEY_GUARD && publicKey
+          ? getBytes(publicKey, curveObject.publicKeyLength)
```

Comment:
How come the passed `publicKey` isn't usable by itself?

Reply:
`publicKey` is typed as `string | Uint8Array`. `getBytes` will just return the `Uint8Array` as-is in this case.

Reply:
So likely this does nothing, assuming we always pass in `publicKey` as `Uint8Array` internally?

Reply:
Yep.

Reply:
Leave a comment please!